src/refer.ts

```diff
@@ -64,6 +90,55 @@ function refer(this: any, options: any) {
     }
   }
 
+  async function actLostEntry(this: any, msg: any) {
+    const seneca = this
+
+    const entryList = await seneca.entity('refer/occur').list$({
+      email: msg.email,
+      kind: 'create',
+    })
+
+    for (let i = 0; i < entryList.length; i++) {
```

Comment:
to support longer term maintainability, let's refactor this. Use Array.filter on entryList to select only the entries that will be updated, then loop over just those entries, removing the conditional from the for loop (`if` statements bad :) ):

```
const chooseAGoodNameForThisFilteredList =
  entryList.filter(entry=>entryList[i].user_id !== msg.userWinner)
```

Reply:
Indeed, thanks for the suggestion.

For this variable name, I choose unacceptedReferrals. What do you think? Changes on commit: 7c1ffb3ae0925129087cd0e22c6e24405f00838d